### ISSUE TITLE
Automatically Reduce Learning Rate

### ISSUE THREAD

**Issue by ztsv-av** [ztsv-av/spellbook#26]
Write autoReduceLR callback (same as Tensorflow ReduceLROnPlateau)

**Comment by ztsv-av** [comment]
done.
